### ISSUE TITLE
Global/Shared Templating

### ISSUE THREAD

**Issue by fabiocosta0305** [hackmdio/hackmd-io-issues#94]
I think something pretty interesting would be someone provide templates that can be shared with other uses.

As a online RPG GM, it would be nice I could provide some templates for Character Sheet and people could use them to create their own characters

**Comment by patcon** [comment]
Agreed. Our group has a hard time using the templates unless they're shareable, because templates are a way that we preserve shared understandings: changes in one person's template must be propagated to others'. Without this, the whole point of the template (for us) is moot.

In my communities, templates are a collab tool, not an individual productivity tool ❤️ 

Specifically, our use-case (perhaps niche) is that we'd like to be able to link to a template from our "how to run a meeting" docs, like so:
https://hackmd.io/9BqEeNuAT7OrWOnZnqXs4A?insert=true&template=0ac83904-f50f-4642-938a-65635edad3f2#

Thanks for any time and consideration you might give this!
